lib/dialects/sqlite3/index.js

```diff
@@ -116,6 +116,12 @@ class Client_SQLite3 extends Client {
     let callMethod;
     switch (method) {
       case 'insert':
+        // if there is any returning column
+        // query should return data
+        if (obj.returning) {
+          callMethod = 'all';
+        }
+        break;
```

Comment:
Friendly suggestion. May want to check the version number on the Sqlite driver object is greater than or equal to `3035000` (i.e. 3.35.0) if so do `RETURNING` but otherwise log the error message. This should allow for support of the feature properly regardless of which version of the Sqlite C library the NPM package was built with 😄

Reply:
Thanks for suggestion I will check if it is needed then implement it